### DISCUSSION TITLE
Help configuring `vim-dadbod-completion`

### DISCUSSION THREAD

**Discussion by eric-2d** [LazyVim/LazyVim#3575] [Q&A] [answered]
Hello,
I'm trying to add [vim-dadbod-completion](https://github.com/kristijanhusak/vim-dadbod-completion)

Looks like all I have to do after adding the plugin is add a line for `nvim-cmp` like so:

```
" hrsh7th/nvim-cmp
  autocmd FileType sql,mysql,plsql lua require('cmp').setup.buffer({ sources = {{ name = 'vim-dadbod-completion' }} })
```

I guess I have to add it to `config/autocmds.lua` however the syntax there is different.

Can you help me translate the line above into lua ?

Thanks

**Comment by dpetka2001** [comment, marked as answer]
There's an Extra about `sql` that also provides this. Have you tried it?

**Comment by eric-2d** [comment]
Oh great I didn't see that, thanks!
